src/clzones.cpp

```diff
@@ -13,7 +13,6 @@
 #include "cursesdef.h"
 #include "debug.h"
 #include "faction.h"
-#include "game.h"
```

Comment:
```
src/clzones.cpp: In member function ‘bool zone_manager::save_zones()’:
src/clzones.cpp:1164:28: error: ‘g’ was not declared in this scope
     std::string savefile = g->get_player_base_save_path() + ".zones.json";
                            ^
src/clzones.cpp: In member function ‘void zone_manager::load_zones()’:
src/clzones.cpp:1177:28: error: ‘g’ was not declared in this scope
     std::string savefile = g->get_player_base_save_path() + ".zones.json";
                            ^
```
https://github.com/CleverRaven/Cataclysm-DDA/pull/42263/checks?check_run_id=886166223#step:4:52

Reply:
Gah, I split this off from #42255 and didn't wait for a full rebuild,